src/limits.cpp

```diff
@@ -104,7 +104,7 @@ void mental_gain(struct char_data * ch)
     gain *= 1.5;
 
   if (GET_TRADITION(ch) == TRAD_ADEPT && GET_POWER(ch, ADEPT_HEALING) > 0)
-    gain *= ((float) 1 + GET_POWER(ch, ADEPT_HEALING) / 2);
+    gain *= ((float) GET_POWER(ch, ADEPT_HEALING) / 2 + 1);
```

Comment:
I see why you moved the +1 to the end of the expression, but could you wrap the get power division portion in parens so it's clear to the reader that we really do intend to do (x/2) + 1 instead of x/(2+1)? I know that it works correctly already due to PEMDAS, this is just a readability thing.